modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/package.scala

```diff
@@ -30,10 +30,12 @@ package engine {
   final case class Action(
     kind: ActionType,
     gen: ActionGen,
-    state: Action.ActionState
+    state: Action.State
   )
   object Action {
 
+    final case class State(runState: ActionState, partials: List[PartialVal])
```

Comment:
Could you add some comments on what the fields are? I have some idea but it's not fully clear